charts/collectorset-controller/Chart.yaml

```diff
@@ -6,7 +6,7 @@ maintainers:
   - email: argus@logicmonitor.com
     name: LogicMonitor
 name: collectorset-controller
-version: 9.3.0-rt01
+version: 9.4.0-rc01
```

Comment:
change it to major version, 10.0.0-rc01

Reply:
@lmshubhashri As discussed in the Slack thread, I've created this PR as a placeholder, and once the code changes are merged to develop. I'll create tags for the major release only.